lib/org/openpsa/invoices/locale/default.en.txt

```diff
@@ -130,6 +130,10 @@ Invoice data
 This invoice has already been sent. Should the file be regenerated anyways?
 ---STRINGEND
 
+---STRING reminder has already been sent. should it be replaced?
+This reminder has already been sent. Should the file be regenerated anyways?
+---STRINGEND
```

Comment:
is this string even used?

Reply:
it is used in:
 $message = $kind . ' has already been sent. should it be replaced?';
 when the $kind variable is removed, this string would be useless. are you sure that i should remove the $kind variable?

Reply:
that line is not reachable if $kind == 'reminder'

Reply:
it can be reached if $kind == 'reminder' and get_status is not 'overdue'

Reply:
that check is wrong anyhow, see the other comment. 

If an invoice is not overdue, there cannot be a reminder by defintion